kiwi/builder/disk.py

```diff
@@ -105,6 +105,13 @@ def __init__(self, xml_state, target_dir, root_dir, custom_args=None):
         self.bootloader = xml_state.build_type.get_bootloader()
         self.initrd_system = xml_state.get_initrd_system()
         self.target_removable = xml_state.build_type.get_target_removable()
+
+        self.root_filesystem_is_multipath = False
```

Comment:
Now I just thought that probably we should do it the other way round, I mean leaving multipath included/untouched in any case and omit it when oem-multipath-scan is set to false, being that the default value.

I am just wondering if we should omit multipath for vmx and isos. Probably it doesn't make much sense to disable it these contexts. Mostly because with this configuration there can't be a live iso or vmx with multipath activated, unless I am missing something.

Reply:
Oh went too fast with the live iso, it should not apply the live iso since it does not use the disk builder, but it is the case for vmx, isn't it?